<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" tests="10" skipped="0" failures="4" errors="0" timestamp="2024-04-03T09:45:08" hostname="LAPTOP-NVVL9JFQ" time="1.258">
  <properties/>
  <testcase name="test_0" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.103"/>
  <testcase name="test_1" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.138"/>
  <testcase name="test_2" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.15"/>
  <testcase name="test_3" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.157">
    <failure message="org.junit.ComparisonFailure: expected:&lt;t[s]itest&gt; but was:&lt;t[i]itest&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;t[s]itest&gt; but was:&lt;t[i]itest&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST.test_3(LONGEST_COMMON_SUBSEQUENCE_TEST.java:30)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:833)
</failure>
  </testcase>
  <testcase name="test_4" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.055"/>
  <testcase name="test_5" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.162">
    <failure message="org.junit.ComparisonFailure: expected:&lt;b[cba]&gt; but was:&lt;b[bdab]&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;b[cba]&gt; but was:&lt;b[bdab]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST.test_5(LONGEST_COMMON_SUBSEQUENCE_TEST.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:833)
</failure>
  </testcase>
  <testcase name="test_6" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.125">
    <failure message="org.junit.ComparisonFailure: expected:&lt;TAAG[]&gt; but was:&lt;TAAG[C]&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;TAAG[]&gt; but was:&lt;TAAG[C]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST.test_6(LONGEST_COMMON_SUBSEQUENCE_TEST.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:833)
</failure>
  </testcase>
  <testcase name="test_7" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.083">
    <failure message="org.junit.ComparisonFailure: expected:&lt;B[CBA]&gt; but was:&lt;B[BDAB]&gt;" type="org.junit.ComparisonFailure">org.junit.ComparisonFailure: expected:&lt;B[CBA]&gt; but was:&lt;B[BDAB]&gt;
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST.test_7(LONGEST_COMMON_SUBSEQUENCE_TEST.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:833)
</failure>
  </testcase>
  <testcase name="test_8" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.086"/>
  <testcase name="test_9" classname="java_testcases.junit.LONGEST_COMMON_SUBSEQUENCE_TEST" time="0.198"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
